### ISSUE TITLE
[windows] The CLI doesn't work at all on Windows with command that require an input from the user

### ISSUE THREAD

**Issue by brandon-welsch** [Scalingo/cli#734]
Log:
```bash
$ scalingo login
        Trying login with SSH
 !      SSH Connection failed.
        Trying login with user/password:

        Username or email: [REDACTED]
        Password:   ! Fail to login (1/3): fail to read the password on stdin: The handle is invalid

        Username or email: interrupt catched, abording
```
And the command doesn't display anything with `mysql-console` for example.

Seems related to https://github.com/99designs/aws-vault/issues/772

Need to test this fix: https://github.com/99designs/aws-vault/pull/113

A customer had this issue: https://app.intercom.com/a/apps/w4oogu7s/inbox/inbox/3379738/conversations/12375700071636
Contact them when this is fixed

**Comment by brandon-welsch** [comment]
From Windows with the Git Bash shell:
```
Brandon@Scalingo MINGW64 ~
$ scalingo.exe login
       Trying login with SSH…
 !     SSH connection failed.
       Trying login with user/password:

       Username or email: [REDACTED]
       Password:  !     Fail to login (1/3): fail to read the password on stdin: Descripteur non valide

       Username or email:
       Username or email:
       Username or email: interrupt catched, aborting…
```

**Comment by johnsudaar** [comment]
Related to [SRE-573]

**Comment by EtienneM** [comment]
Another impacted customer: https://app.intercom.com/a/inbox/w4oogu7s/inbox/shared/all/conversation/12375700080025

**Comment by EtienneM** [comment]
This will be fixed for the next release including the fix in https://github.com/Scalingo/cli/pull/806. Probably 1.26.0.

**Comment by EtienneM** [comment]
FYI the fix from https://github.com/99designs/aws-vault/pull/113 has been implemented in the PR #705:

https://github.com/Scalingo/cli/pull/705/files#diff-40837a1735f9474e15525ba28f225c256a9ab77d7c5f1fce60caa79e7a33243bR13
